plugins/MauticSocialBundle/Command/MonitorTwitterBaseCommand.php

```diff
@@ -245,6 +253,38 @@ protected function createLeadsFromStatuses($statusList, $monitor)
                     ],
                 ]
             );
+            if ($monitorProperties['matchnames'] === 1) {
```

Comment:
This line checks for a property called `matchnames`, but in the DB, the property is stored as `checknames`:
```
'a:2:{s:7:"hashtag";s:9:"TraficoGT";s:10:"checknames";i:1;}'
```